source/community/reactnative/src/components/containers/column.js

```diff
@@ -64,120 +64,7 @@ export class Column extends React.Component {
 	 * @returns {boolean} true|false
 	 */
 	isForemostElement = () => {
```

Comment:
Spacing and Separator are missed and not working as expected. (Example: Refer `flight-update.json` sample in Visualizer)

Reply:
Resolved.